ui/app/models/clients/version-history.js

```diff
@@ -0,0 +1,5 @@
+import Model, { attr } from '@ember-data/model';
+export default class VersionHistoryModel extends Model {
+  @attr('array') keys;
+  @attr('object') keyInfo;
```

Comment:
I wonder if it would be valuable to store each version upgrade data as a new record 🤔 especially for the future where we may incorporate these values into the charts